cmd/filesystem.go

```diff
@@ -564,7 +564,7 @@ func init() {
 				IsSet: true,
 				Th: check.Threshold{
 					Lower: 2,
-					Upper: 100,
```

Comment:
Can you add a bit more context on why? Intuitively I would have said a percentage - having a Latin "per cento" origin meaning "from 100" - would have 100 as upper.

Reply:
That is indeed true and was my original thought here. The problem is, that some Monitoring systems (Icinga) chose to ignore the standard and are unable to parse the range format (`2:100`), so I replaced that here with infinity which results in the range of `2:` which can be parsed and is functionally equivalent.
Tl:dr: It's a hack.